HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json

```diff
@@ -80,9 +80,8 @@
       "item": [
         {
           "linkId": "4.1",
-          "text": "Diabetes Mellitus",
+          "text": "Diabetes Mellitus Diagnoses",
```

Comment:
This PR was supposed to focus on the HBGM Order. Why are there changes to the F2F questionnaire here, @yuliumitre?

Reply:
This change is triggered by the discussion about "Diagnoses". Original thoughts was to change HBGM order and F2F in this branch as a model. This change just makes the names consistent across order and F2F form.  I also have another branch to address the issues for other topics.